Makefile

```diff
@@ -776,8 +776,7 @@ docopen: $(apidocs_html)
 
 .PHONY: docserve
 docserve: $(apidocs_html)
-	@$(PYTHON) -mwebbrowser http://localhost:8000/all.html
```

Comment:
Why is this line removed? I find it handy to have the browser open when running the command.

Reply:
If I had not removed this line, I would have continued recieving the same error seen in the origin issue.

Reply:
Interesting, for me it pulls up the docs in my default browser. I just realised Python server put a link to the newly created webserver that I can Ctrl+click so it's even better.